### PR TITLE
ci: update release-please action and config

### PR DESCRIPTION
**Pull request by jokerttu**
Update release please CI action script and configurations

### CODE REVIEW

File: .release-please-manifest.json
Comment (wangela) [inline .release-please-manifest.json:1]: Let's establish that the initial version for this manifest is 0.4.0, similar to https://github.com/googlemaps/google-maps-ios-utils/blob/main/.release-please-manifest.json

File: release-please-config.json
Comment (wangela) [inline release-please-config.json:1]: See https://github.com/googlemaps/google-maps-ios-utils/blob/main/release-please-config.json for example config.

- Bootstrap SHA should be the commit related to the 0.4.0 release, i.e. 19d2491ea1a03add016b31b94562da03ccefaa74
- Based on [configfile README](https://github.com/googleapis/release-please/blob/main/docs/manifest-releaser.md#configfile) I agree `bump-minor-pre-major` should be true for this pre-1.0 package
- Let's raise `bump-minor-pre-major` should be one level to be same level as `include-component-in-tag`
- Specify the `changelog-path` for the package

**Review by wangela — CHANGES_REQUESTED** [review]

**Review by caio1985 — APPROVED** [review]
LGTM

**Review by illuminati1911 — APPROVED** [review]
LGTM

**Pull request merged**